src/EditorFeatures/Core/Implementation/MetadataAsSource/MetadataAsSourceFileService.cs

```diff
@@ -67,7 +67,7 @@ private string GetRootPathWithGuid_NoLock()
             return _rootTemporaryPathWithGuid;
         }
 
-        public async Task<MetadataAsSourceFile> GetGeneratedFileAsync(Project project, ISymbol symbol, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<MetadataAsSourceFile> GetGeneratedFileAsync(Project project, ISymbol symbol, OptionSet optionSet = null, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
Do we need it to be explicit, or can you just pick up the options from the Project that's already being passed in?